src/optimization/convDual.jl

```diff
@@ -72,6 +72,17 @@ function backprop!(v::Vector{Float64}, u::Vector{Float64}, l::Vector{Float64})
     return o
 end
 
+function get_bounds(solver::ConvDual, nnet::Network, input::Hyperrectangle)
+    # only supports hypercubes in this implementation
+    @assert all(input.radius[1] .== input.radius)
+    L, U = get_bounds(nnet, input.center, input.radius[1])
+    # add the bounds from the input set
+    pushfirst!(L, low(input))
+    pushfirst!(U, high(input))
+    # convert convdual's bounds into hyperrectangles
+    return [Hyperrectangle(low=L[i], high=U[i]) for i = 1:length(L)]
+end
```

Comment:
This doesn't yet stick to the new `get_bounds` interface, since it doesn't support `before_act=true/false`. Is there a way that it could? If not, perhaps just adding the keyword as a dummy, and then asserting that it is `false` would make it at least behave the same.